src/lib/components/documents/ResultsList.svelte

```diff
@@ -177,6 +186,12 @@
 
   .end {
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    gap: 0.5rem;
+  }
+
+  .error {
+    color: var(--error, red);
```

Comment:
Since we're showing text stacked in a centered container, let's center it, too:

```suggestion
    text-align: center;
    color: var(--error, red);
```